src/controllers/userController.js

```diff
@@ -43,6 +44,22 @@ const signUp = async (req, res) => {
   }
 };
 
+const logout = async (req, res) => {
+
+  const redis = new Redis(process.env.REDIS_URL)
+  const token = req.headers.authorization.split(' ')[1]
+
+  try {
+    await redis.set(token, token, 'EX', 604800000)
```

Comment:
is there a reason token is used twice here?

Reply:
I dont wont to leave the value parameter enter, the key is whats needed. Should i change the value to an empty string?